pppp/src/views/pay-practitioner/MainFormPage.vue

```diff
@@ -504,32 +585,48 @@ export default {
         required,
         phnValidator,
       },
+      dependentNumber: {
+        intValidator: optionalValidator(intValidator),
+        positiveNumberValidator: optionalValidator(positiveNumberValidator),
+      },
       firstName: {
         required,
+        nameValidator,
+      },
+      middleInitial: {
+        nameInitialValidator,
       },
       lastName: {
         required,
+        nameValidator,
       },
       birthDate: {
         required,
       },
       isVehicleAccident: {
         required,
       },
+      vehicleAccidentClaimNumber: {
+        intValidator: optionalValidator(intValidator),
+        positiveNumberValidator: optionalValidator(positiveNumberValidator),
+      },
       medicalServiceClaims: {
         $each: {
```

Comment:
You'll need to add a `maxLength` validator to the `notes` model. See Pay Patient as an example:

https://github.com/bcgov/MOH-PPPP-Enrolment/blob/main/pppp/src/views/pay-patient/MainFormPage.vue#L598-L600

Can skip adding an error message since the "Remaining characters text in the `Textarea` component turn red when the user is over the limit.

Same for `hospitalVisitClaims` notes.